### ISSUE TITLE
Shantae unknown .image

### ISSUE THREAD

**Issue by makc-ar** [meh2481/MSFHDEx#2]
Example: https://mega.nz/#!EpQGTQiC!Z_ZM-wqYptqfSoZguT1q-cYchb417ZdRzTUexiyk6Ic
How about packing .image?

**Comment by meh2481** [comment]
Yeah, I dunno what's up with those. It looks like a variation on the format, but from a glance-over it's not immediately obvious to me where the differences lie.
